src/entry.py

```diff
@@ -209,6 +209,9 @@ def process_files(
     for file_path in omr_files:
         files_counter += 1
         file_name = file_path.name
+        evaluation_path = os.path.join(
+            outputs_namespace.paths.evaluation_dir, file_path.stem
```

Comment:
let's directly pass `evaluation_output_dir` (= outputs_namespace.paths.evaluation_dir) into the function below